keras_retinanet/preprocessing/pascal_voc.py

```diff
@@ -77,7 +77,7 @@ def __init__(
         data_dir,
         set_name,
         classes=voc_classes,
-        image_extension='.jpg',
+        image_extension,
```

Comment:
It's not allowed to have a positional argument after optional arguments.

Reply:
Image extension must be before to classes, right?

Reply:
```python
def foo(a, b=1, c):
	print(a, b, c)

foo(1, 2)
```

```shell
  File "test.py", line 1
    def foo(a, b=1, c):
            ^
SyntaxError: non-default argument follows default argument
```

You can give the `image_extension` a default value, making it an optional argument.

Reply:
Ok,